test/iscsi_client_lib_test.rb

```diff
@@ -0,0 +1,75 @@
+require_relative "test_helper"
+
+require_relative "../src/modules/IscsiClientLib"
+
+describe Yast::IscsiClientLibClass do
+  subject { described_class.new }
+
+  describe "#getServiceStatus" do
+    before do
+      allow(Yast::Service).to receive(:Stop).with(anything)
+      allow(Yast::Service).to receive(:active?).with(anything)
+      allow(Yast::SystemdSocket).to receive(:find!).with(anything)
+      allow(subject).to receive(:socketActive?)
+      allow(subject.log).to receive(:error)
+    end
+
+    context "for iscsid.socket" do
+      let(:iscsid_socket) { double("iscsid.socket", start: true) }
+
+      before do
+        allow(Yast::SystemdSocket).to receive(:find!).with("iscsid").and_return(iscsid_socket)
+        allow(subject).to receive(:socketActive?).with(iscsid_socket).and_return(socket_status)
+      end
+
+      context "when iscsid.socket and iscsid.service are active" do
+        let(:socket_status) { true }
+
+        before do
+          allow(Yast::Service).to receive(:active?).with("iscsid").and_return(true)
+        end
+
+        it "does nothing" do
```

Comment:
I would say something like *does not start the service nor the socket" because "does nothing" says nothing to me :)